awx/ui_next/src/components/JobList/JobList.jsx

```diff
@@ -238,6 +238,13 @@ function JobList({ i18n, defaultParams, showTypeColumn = false }) {
                   onDelete={handleJobDelete}
                   itemsToDelete={selected}
                   pluralizedItemName={i18n._(t`Jobs`)}
+                  cannotDelete={item =>
+                    item.status === 'running' ||
```

Comment:
There's probably more than one status we'll need to check for. Let's use the `isJobRunning` util: https://github.com/ansible/awx/blob/devel/awx/ui_next/src/util/jobs.js#L10

Reply:
I will also try to use the plural for the error message.

Reply:
@jakemcdermott, updated.